src/v0/destinations/klaviyo/transform.js

```diff
@@ -186,8 +189,9 @@ const trackRequestHandler = (message, category, destination) => {
       ...attributes.properties,
       ...populateCustomFieldsFromTraits(message),
     };
-    attributes.profile = createCustomerProperties(message);
   }
+  // Map user properties to profile object
+  attributes.profile = createCustomerProperties(message, destination.Config);
```

Comment:
Is it customerProperties or customProperties ?

Reply:
These are customer's profile properties. Eg. Email, firstname, lastname etc